srcpkgs/xbps-triggers/template

```diff
@@ -1,19 +1,23 @@
 # Template file for 'xbps-triggers'
 pkgname=xbps-triggers
-version=0.110
+version=0.111
 revision=1
 noarch=yes
 bootstrap=yes
 short_desc="The XBPS triggers for Void Linux"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="http://www.voidlinux.eu"
-license="Simplified BSD"
+license="BSD-2-clause"
 
 do_install() {
 	_triggersdir=usr/libexec/${pkgname}
 
 	for f in ${FILESDIR}/*; do
 		_trigger=$(basename $f)
+		if test "${_trigger}" = "LICENSE"; then
```

Comment:
i'd rather just download the license with XBPS_FETCH_CMD

Reply:
from where should I download it? if you have an example that would be nice.

Reply:
Looking at it more i think it is better we just don't worry about it

Reply:
so leave as is, or leave the lint